automated_packaging/prepare_changelog.pl

```diff
@@ -0,0 +1,60 @@
+#!usr/bin/perl
+$CODE_PATH=$ARGV[0];
+$VERSION=$ARGV[1];
+$PROJECT=$ARGV[2];
+$SHA_CODE_OF_LAST_COMMIT=$ARGV[3];
+
+# Name of the repo is represented differently on logs and repos
+$repo_log_name="citus";
+$master_branch_name="master";
```

Comment:
I think it would be more appropriate to get the name of the branch from user. In citus-enterprise we use 'enterprise-master' instead of 'master'. Even better you can use 'master' as default value if user does not specify branch name.